base/uk.ac.stfc.isis.ibex.scriptgenerator.tests/src/uk/ac/stfc/isis/ibex/scriptgenerator/tests/ScriptGeneratorSingletonTest.java

```diff
@@ -156,53 +153,4 @@ public void test_GIVEN_new_file_WHEN_generate_THEN_filename_without_version() {
 		}
 	}
 	
-	@Test
```

Comment:
Where have these tests been moved to?

Reply:
I think I had meant to reimplement them somewhere but somehow forgot? Not sure. Anyway they've been added now one of them in here and multiple more in ScriptGeneratorFileHandlerTest.java